jmbitcoin/jmbitcoin/secp256k1_transaction.py

```diff
@@ -534,23 +534,29 @@ def pubkey_to_p2wpkh_address(pub):
     script = pubkey_to_p2wpkh_script(pub)
     return script_to_address(script)
 
-def pubkeys_to_p2wsh_script(pubs):
+def pubkeys_to_p2wsh_multisig_script(pubs):
     """ Given a list of N pubkeys, constructs an N of N
     multisig scriptPubKey of type pay-to-witness-script-hash.
     No other scripts than N-N multisig supported as of now.
```

Comment:
Probably edit this comment I guess.

Reply:
To say what?

Reply:
Ah sorry, slightly obtuse comment; I mean " No other scripts than N-N multisig supported as of now." is no longer relevant here now as it's been renamed to be exactly that.